pkg/erofs/erofs.go

```diff
@@ -495,11 +499,9 @@ func ExtractSingleErofsPolicy(erofsFile, extractDir string, policy *ExtractPolic
 // wik()th that.
 func ExtractSingleErofs(erofsFile string, extractDir string) error {
 	exPolInfo.once.Do(func() {
-		const envName = "STACKER_EROFS_EXTRACT_POLICY"
-		const defPolicy = "kmount erofsfuse fsc.erofs"
-		val := os.Getenv(envName)
+		val := os.Getenv(PolicyEnvName)
```

Comment:
To differentiate between an empty value and an unset variable, os.LookupEnv can be used.

Reply:
What would we want to do differently between empty or undefined?  If they really meant "empty", then they can't use this tool at all...

Reply:
So I think we don't care to distinguish.  Please re-open this thread if you think there should actually be different behavior.